arctic/store/_ndarray_store.py

```diff
@@ -129,9 +126,12 @@ def can_delete(self, version, symbol):
     def can_read(self, version, symbol):
         return version['type'] == self.TYPE
 
-    def can_write(self, version, symbol, data):
+    def can_write(self, version, symbol, data, **kwargs):
         return isinstance(data, np.ndarray) and not data.dtype.hasobject
 
+    def can_update(self, version, symbol, data, **kwargs):
```

Comment:
Can we push this default implementation up into a generic base-class?  Our TimeSeries chunk store (derived from `VariableChunkStore`) will also need changing, so it should probably inherit basic functionality from a generic `ABC`